pitrery

```diff
@@ -3688,6 +3688,9 @@ case $action in
 				else
 					ts=$(ssh -n -- "$backup_ssh_target" "cat -- $(qw "$dir/backup_timestamp")")
 				fi
+				if [ "$?" -ne "0" ]; then
```

Comment:
Quotes around 0 are useless. We currently use != instead of -ne  to compare return code.

Reply:
Fixed

Reply:
great :)